plugin.rb

```diff
@@ -135,63 +139,36 @@ def fetch_user_details(token, id)
       nil
     end
   end
+  
+  def primary_email_verified?(auth)
+    ActiveModel::Type::Boolean.new.cast(auth['info']['email_verified']) ||
```

Comment:
I'm curious why this was required. Do you have an authentication system which was returning "true" as a string, rather than a native boolean?

Reply:
tbh I can't recall, but yes, that was what this would have been guarding against. We could take it out, but do you feel it poses a risk leaving it in?

Reply:
I've taken out the cast :+1: